example/py-shiny/example_pyshiny_reactive_scatter.py

```diff
@@ -0,0 +1,57 @@
+import matplotlib.pyplot as plt
+import seaborn as sns
+from shiny import App, reactive, render, ui
+
+from maidr.custom_renderers.shiny_renderer import render_maidr
+
+# Load the dataset
+iris = sns.load_dataset("iris")
+
+# Define the UI components for the Shiny application
+app_ui = ui.page_fluid(
+    ui.row(
+        ui.column(
+            3,
+            ui.input_select(
+                "x_var",
+                "Select X variable:",
+                choices=iris.select_dtypes(include=["float64"]).columns.tolist(),
+                selected="sepal_length",
+            ),
+            ui.input_select(
+                "y_var",
+                "Select Y variable:",
+                choices=iris.select_dtypes(include=["float64"]).columns.tolist(),
+                selected="sepal_width",
+            ),
+        ),
+        ui.column(9, ui.output_ui("reactivebarplot")),
+    )
+)
+
+
+# Define the server
+def server(input, output, session):
+    @reactive.Calc
```

Comment:
`reactive.Calc` is mainly used to perform mathematical intensive calculations. May I know why this has been used here?